rules/hocon.bzl

```diff
@@ -0,0 +1,89 @@
+"""Lucid Hocon compilation rules
+
+Bazel rules for compiling multiple HOCON source files into a flattend HOCON artifact.
+
+In addition to checking syntax and flattening the configuration files, it also verifies that any references to
+lucidBag refer to keys that are available for the applicable IAM roles
+
+[HOCON](https://github.com/lightbend/config)
+
+"""
```

Comment:
Is `lucidBag` also open source? Seems strange to reference it here, if not.

Reply:
good catch